src/gc/proxy.d

```diff
@@ -76,12 +76,30 @@ extern (C)
             }
             _instance = newInstance;
             // Transfer all ranges and roots to the real GC.
-            (cast(ProtoGC) protoInstance).term();
+            (cast(ProtoGC) protoInstance).transfer();
             isInstanceInit = true;
         }
         instanceLock.unlock();
     }
 
+    // Call this function when initializing the real GC.
+    // This function should be called after the real GC is in place.
+    private void transfer(ProtoGC src)
```

Comment:
Moving this function exposes internals of ProtoGC here. I don't think that's desirable.